### PR TITLE
Terraform

### PR DESCRIPTION
**Pull request by slagathorr**
Tweaked a couple things based off of Jennie's comments on the last PR#22, but I didn't know how (or if it's even possible) to merge that into an open PR, so I just closed it, pushed the changes, and opened a new PR. If there's a better way to do that, I'd love to know!

### CODE REVIEW

**Review by mayaspivak — CHANGES_REQUESTED** [review]
Are the test datafiles used in this PR? They seem out of place. Maybe move to a different PR?

**Comment by slagathorr** [comment]
> Are the test datafiles used in this PR? They seem out of place. Maybe move to a different PR?

@mayaspivak  For this build they're used to load the BQ tables and the SQL query won't work without them. It's intended to be a temporary placeholder so there's a structure to build off of, and will be deleted and redirected to the dynamically generated tables once the pipelines are in place.

**Comment by slagathorr** [comment]
@mayaspivak  Okay I'm still learning how to github, wasn't sure how to comment directly on your change request, so hit the "re-request review" button but not sure if that's the right workflow.

**Comment by jenniebrown** [comment]
> Tweaked a couple things based off of Jennie's comments on the last PR#22, but I didn't know how (or if it's even possible) to merge that into an open PR, so I just closed it, pushed the changes, and opened a new PR. If there's a better way to do that, I'd love to know!

If you commit and push your changes, they should be automatically added to the PR so you can just hit 're-request review' 
(found the answer from this article: https://homes.cs.washington.edu/~mernst/advice/github-pull-request.html#respond-to-reviews)

**Comment by slagathorr** [comment]
> > Tweaked a couple things based off of Jennie's comments on the last PR#22, but I didn't know how (or if it's even possible) to merge that into an open PR, so I just closed it, pushed the changes, and opened a new PR. If there's a better way to do that, I'd love to know!
> 
> If you commit and push your changes, they should be automatically added to the PR so you can just hit 're-request review'
> (found the answer from this article: https://homes.cs.washington.edu/~mernst/advice/github-pull-request.html#respond-to-reviews)

Ah okay, that's helpful to know. And also makes WAY more sense. Also I've been using PRs incorrectly (when to use them and how to logically group changes and such) so good to know to make them more readable going forward. Thanks!

File: prototype.tf [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+# Use the ./sql/pdccr.sql file
+resource "google_bigquery_job" "bq_job_pdccr_ucf_joined" {
+  depends_on = [google_bigquery_job.bq_job_load_pdccr, google_bigquery_job.bq_job_load_ucf]
+  job_id     = "pdccrucfjoin${formatdate("YYYYMMDDhhmmss",timestamp())}"
+
+  query {
+    query = file("./sql/pdccr.sql")
+
+    destination_table {
+      table_id = google_bigquery_table.bqt_pdccrucfjoin.id
```
Comment (jenniebrown) [inline prototype.tf]: Does this name need to match the name of the resource (i.e. bqt_pdccr_ucf_joined instead of bqt_pdccrucfjoin)?

Comment (slagathorr) [reply prototype.tf]: Yep, it does. Was a typo from fixing the comments, just tweaked and re-ran the script to test that resources deploy successfully.

**Review by jenniebrown — APPROVED** [review]
LGTM but be sure to wait for Maya's input before submitting (I'm not sure if my approving this will negate her requested changes in the UI)

**Review by mayaspivak — APPROVED** [review]

**Pull request merged**